Add -y option to `zpool iostat`

dasjoe opened the pull request:
sysstat's iostat omits the first report when the -y option is used.
This patch adds that functionality and omits the first report with
statistics since system boot.

behlendorf commented:
Nice enhancement.  Interestingly enough `iostat(1)` doesn't have a `-y` option, this seems to be an illumos thing.  But `-y` seems like an entirely reasonable flag name to me.

dasjoe commented:
@behlendorf I took `-y` from sysstat's iostat, it has been added in version 10.1.3.
See https://github.com/sysstat/sysstat/blob/master/CHANGES#L284-L286 for "upstream's" changelog.

I reworked the commit and changed the way skip was set, according to your suggestion.

behlendorf commented:
Merged with the small addition of an entry in `zpool(8)` describing the option.

@dasjoe any interest in adding a `-x` option as well like `iostat(1)`?
